skins/laika/src/components/shared/Postal.vue

```diff
@@ -42,45 +42,78 @@
 		<span v-if="showError.city" class="help is-danger">{{ $t('donation_form_city_error') }}</span>
 		<span v-if="cityValueEqualsPlaceholder" class="help">{{ $t('donation_form_city_placeholder_warning') }}</span>
 	</div>
-	<div class="form-input">
+
+	<div v-bind:class="['form-input', { 'is-invalid': showError.country }]">
 		<label for="country" class="subtitle">{{ $t( 'donation_form_country_label' ) }}</label>
-		<b-field>
-			<b-select
+		<b-field :type="{ 'is-danger': showError.country }">
+			<b-autocomplete
 					class="is-form-input"
-					v-model="formData.country.value"
-					id="country"
+					field="countryFullName"
+					placeholder="z. B. Deutschland"
+					v-model="countryInput"
 					name="country"
-					@blur="$emit('field-changed', 'country')">
-				<option v-for="(countryCode, index) in countries"
-						:value="countryCode"
-						:key="index">{{ $t('donation_form_country_option_' + countryCode ) }}
-				</option>
-			</b-select>
+					id="country"
```

Comment:
`id` is probably a vestige that can be removed

Reply:
It's used for the label targeting